src/Microsoft.DotNet.Watcher.Core/Internal/FileWatcher/PollingFileWatcher.cs

```diff
@@ -0,0 +1,243 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+
+namespace Microsoft.DotNet.Watcher.Core.Internal
+{
+    internal class PollingFileWatcher : IFileSystemWatcher
+    {
+        // The minimum interval to rerun the scan
+        private static readonly TimeSpan _minRunInternal = TimeSpan.FromSeconds(.5);
```

Comment:
Now that this is inside dotnet-watch shouldn't all the configuration options be passable as both env and command line args?